### PR TITLE
Maximum File Length

### PR DESCRIPTION
**Pull request by lowtorola**
Not sure how to test this implementation of maximum file length so this is a draft PR for now... would appreciate some eyes on this.

### CODE REVIEW

**Comment by lowtorola** [comment]
Need to test! Gaurav suggested using this on avatars to test so will try this.

**Comment by j-mao** [comment]
Haven't tried it myself, but looks like it works! One of the test cases generates an awfully large file upload, and that test case now fails :P

**Comment by gaurav-arya** [comment]
By the way your commits still seem to not be associated with your GitHub account; might be worth fixing that and rebasing:)

**Comment by lowtorola** [comment]
> By the way your commits still seem to not be associated with your GitHub account; might be worth fixing that and rebasing:)

That's odd, they seem to be associating now on my screen. Before I "fixed" it they didn't have my profile pic but now they do?

**Comment by gaurav-arya** [comment]
hm, on my end your commits still don't have a profile pic (e.g. n the commits tab of this page)

**Comment by lowtorola** [comment]
> hm, on my end your commits still don't have a profile pic (e.g. n the commits tab of this page)

Ah, I see. I'm going to try to fix that at the meeting Saturday :+1:

**Comment by gaurav-arya** [comment]
Hmm, strange, I don't see a profile pic on your commits on my end

File: backend/siarnaq/api/compete/serializers.py
Comment (gaurav-arya) [inline backend/siarnaq/api/compete/serializers.py:94]: I think it's confusing to have this as a class variable if it's only needed for defining the line below. Happy if you just inline this into the below line, or any other alternate solution where we don't have an extra class variable

**Review by gaurav-arya — DISMISSED** [review]
Looks good to me, other than the comment I left! But wait for an additional review from @j-mao

File: backend/siarnaq/api/compete/serializers.py
Comment (j-mao) [reply backend/siarnaq/api/compete/serializers.py:94]: Agree with this- otherwise all looks good to me!

**Comment by lowtorola** [comment]
Looks like the change has to be approved before I can merge :rocket:

**Review by gaurav-arya — DISMISSED** [review]
Done! Please make sure to give the latest version a quick test in staging, then feel free to merge :)

**Comment by lowtorola** [comment]
@gaurav-arya It seems to want this PR to pass all of the checks before it will let me merge... is there a workaround for this? Kinda funny because this PR intentionally fails the "large file submission" Django test Jerry mentioned earlier.

**Comment by j-mao** [comment]
Hahaha good point. I think that test case is now bad because it… isn't supposed to work anymore :P

I'll push a change to this test that makes it smaller :')

**Review by j-mao — APPROVED** [review]

**Pull request merged**
